addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -129,8 +129,13 @@ export class MassMailingHtmlField extends HtmlField {
             // Wait for the css and images to be loaded.
             await iframePromise;
             const editableClone = iframe.contentDocument.querySelector('.note-editable');
-            this.cssRules = this.cssRules || getCSSRules($editable[0].ownerDocument);
-            await toInline($(editableClone), this.cssRules, $(iframe));
+            // editableClone doesn't have wysiwyg in its data
+            // we use this hacky fix to assign it now so we can get it later in toInline
+            // from the editable
+            // TODO in master : Update the function toInline to accept an options dictionary
+            // as paraemeter.
```

Comment:
Reworded to wrap at 80 chars:
```suggestion
            // The jQuery data are lost because of the cloning operations above.
            // The hacky fix for stable is to simply add it back manually.
            // TODO in master: Update toInline to use an options parameter.
```
Other than that, great catch ! 👍